src/core/services/layout/layout.js

```diff
@@ -42,25 +44,41 @@
      */
     angular.module('material.core.layout', [ 'ng' ])
 
+      /**
+       * Model of flags used by the Layout directives
+       * Allows changes while running tests or runtime app changes
+       */
+      .factory("$$mdLayout", function() {
+        return {
+          /**
+           * Should we remove the original layout Attribute selectors
```

Comment:
I think this should be a provider, since we only want to configure this 1x only or in tests.
Your thoughts ?